File: 1713/1713.c
int minimumOperationsToMakeASubseq(struct TreeNode* root) {
    if (!root) return 0;
    int l = minimumOperationsToMakeASubseq(root->left);
    int r = minimumOperationsToMakeASubseq(root->right);
    return 1 + (l > r ? l : r);
}

File: 1713/1713.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var minimumOperationsToMakeASubseq = function(root) {
    if (!root) return 0;
    return 1 + Math.max(minimumOperationsToMakeASubseq(root.left), minimumOperationsToMakeASubseq(root.right));
};